### PR TITLE
Add courgette fritters recipe

### PR DESCRIPTION
**Pull request by m4v15**
relates #9 

Only to be merged by @sofer, @mattlub or @Karyum

### CODE REVIEW

**Review by ZooeyMiller — APPROVED** [review]
looks awesome :plate_with_cutlery: :tada:

can't wait to eat this :wink:

**Review by yvonne-liu — COMMENTED** [review]
A photo of the dish would be v nice 🥒 ✨

**Review by yvonne-liu — APPROVED** [review]

**Review by finnhodgkin — COMMENTED** [review]
:heart: perfect

**Review by finnhodgkin — APPROVED** [review]

File: recipes/main/courgette_arabic_cheese_fritters.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Courgette Fritters 
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-70%25-orange.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-1hr-yellow.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Cheap-green.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-False-red.svg)
+
+**First Cooked by:** @m4v15
```
Comment (yvonne-liu) [inline recipes/main/courgette_arabic_cheese_fritters.md:9]: [@m4v15](https://github.com/m4v15) 😉

**Review by yvonne-liu — CHANGES_REQUESTED** [review]
Fix yo markdowns :fire:

File: recipes/main/courgette_arabic_cheese_fritters.md (hunk 1 above)
Comment (m4v15) [reply recipes/main/courgette_arabic_cheese_fritters.md:9]: Done 😃

File: recipes/main/courgette_arabic_cheese_fritters.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Grate or finely slice the courgettes, and place in a mixing bowl with some salt to draw out the water.
+2. Finely dice the onions and garlic, add to drained courgettes (squeeze them to get as much water out as possible).
+3. Add beaten eggs and flour, as much flour as you need until you get quite a thick batter.
+4. Add spices and seasoning, to taste.
+5. Dice cheese into 1cm cubes, add as much or as little as you like.
+6. Heat a fair amount of oil in a pan (2cm deep) to smoking point, add big spoonful of batter mix, fry until golden brown and flip (4-5 minutes on each side).
+7. Keep warm in oven whilst cooking all of the mix.
+
+![Picture of 2 happy diners](https://user-images.githubusercontent.com/23310908/28714486-4e2e9fea-739c-11e7-8e57-96e50a149ed7.jpg)
```
Comment (Karyum) [inline recipes/main/courgette_arabic_cheese_fritters.md:37]: Beautiful :laughing:

**Review by Karyum — APPROVED** [review]
looks delicious

**Pull request merged**